docs/maya/glue.md

```diff
@@ -0,0 +1,145 @@
+# AdnGlue
+
+AdnGlue is a Maya node that can be used to attach multiple muscles together. This solver uses as input a series of simulated muscles and allows the user to define glue connections between those muscles by using a maximum distance value and painted maps.
+
+The maps required to control the AdnGlue solver can be painted using the standard Maya Paint tool.
+
+### How To Use
+
+The AdnGlue node can be applied to any set of geometries. For each geometry, their mesh at each frame is gathered and simulated to ensure the original distance between the neighboring geometries. This is achieved by creating attachments between each vertex of a geometry and the closest point on the surface of the other geometries. More aspects are then computed to ensure that the simulated output preserves the original shape, edge-lengths and volume of the simulated meshes. The output of the node is a mesh that combines all the input geometries together and adds the results of the solver.
```

Comment:
"volume of the simulated meshes" if maybe a user reads this and does see that the glue is not preserving volume they might think tha the volume constraint is not working when it's actually not integrated. This is just a thought, this sentence still makes sense!

Reply:
I agree. I have removed "and volume"